ui/lib/core/addon/components/certificate-card.hbs

```diff
@@ -20,13 +20,13 @@
       {{@data}}
     </code>
   </div>
-  <div class="is-flex-center has-background-white-bis has-side-padding-s has-top-bottom-margin-negative-m">
+  <div class="is-flex-center has-background-white-bis has-top-bottom-margin-negative-m">
```

Comment:
It looks like some styling changes to this component caused weird button padding here
# after upgrading to `3.1.0`
<img width="425" alt="Screenshot 2023-12-04 at 3 48 10 PM" src="https://github.com/hashicorp/vault/assets/68122737/ad5fd0ec-1000-4433-a59d-bfacd10db19c">

# addressed by removing side badding and now:
<img width="418" alt="Screenshot 2023-12-04 at 3 47 22 PM" src="https://github.com/hashicorp/vault/assets/68122737/8e315dd6-a247-4adc-9525-a03b0f3455b9">